packages/relay/src/lib/eth.ts

```diff
@@ -147,7 +147,13 @@ export class EthImpl implements Eth {
   }
 
   private async getFeeWeibars() {
-    let networkFees = await this.mirrorNodeClient.getNetworkFees();
+    let networkFees;
+
+    try {
+      networkFees = await this.mirrorNodeClient.getNetworkFees();
+    } catch (e: any) {
+      this.logger.debug(`Failed to retrieve Network fees from Mirror Node`);
```

Comment:
Should be warn level if an upstream has an error.

Reply:
Updated to warn